.meta/template/globals.lua

```diff
@@ -284,6 +280,12 @@ function physics.garbageCollectBullets() end
 ---@param flags integer
 function physics.createBlock(blockX, blockY, blockZ, flags) end
 
+---Get the the collidable block in the level.
+---@param blockX integer
+---@param blockY integer
+---@param blockZ integer
+function physics.getBlock(blockX, blockY, blockZ) end
```

Comment:
1. This function returns the flags of the block at the specified coordinates.
2. No return value assigned, use 'itneger'